grails-app/controllers/org/pih/warehouse/LoggingInterceptor.groovy

```diff
@@ -31,19 +38,21 @@ class LoggingInterceptor {
             MDC.put('queryString', request?.queryString ?: "No query string")
             MDC.put('serverUrl', Holders.grailsApplication.config?.grails?.serverURL ?: "No server URL")
         } catch (Exception e) {
-            LoggingInterceptor.log.warn("Error occurred while adding attributes to Mapped Diagnostic Context: ${e.message}", e)
-
+            log.warn("Error occurred while adding attributes to Mapped Diagnostic Context: ${e.message}", e)
         }
+
+        // Now that we've populated the MDC, log the request itself, including query parameters.
+        String queryString = StringUtils.isBlank(request?.queryString) ? "" : "?${request?.queryString}"
+        log.info("${request.requestURI}${queryString} [user:${session?.user?.username}, location:${session?.warehouse?.name}]")
```

Comment:
I think this is the log message that @awalkowiak doesn't like. I find it helpful to demarcate logging for requests along with the message that reports response time. 

Aside: Now that I'm looking at the logs I'm noticing that the response time is no longer reported. @awalkowiak Do you know when that got removed? 

Anyway, I can be overruled if others would prefer to remove it or make it a DEBUG message.

Reply:
@jmiranda this one I like. I often look at this one. The one I dislike that is causing a lot of garbage is the one in the `UserService` -> `log.info "Is role ${role.roleType} in ${acceptedRoleTypes} = ${acceptedRoleType}"` (and I am happy that it was demoted to debug level)